### PR TITLE
Tuesday January 11 2022 Meeting

### PR DESCRIPTION
**Pull request by nkohen**
Please comment on this PR or DM with things we should discuss or other changes that should be made to this document.

Once the meeting has happened I will add meeting notes and then once they have been reviewed this will be merged into meetings directory.

### CODE REVIEW

File: meeting/meeting_january_11_2022.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* [Paper!](https://github.com/Tibo-lg/dlcspecpaper/blob/v0/dlcpaper.pdf)
+  * **#Query(Tibo)**
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* [Numerical Decomposition Clarification](https://github.com/discreetlogcontracts/dlcspecs/pull/182)
+  * Needs Review
+* Using `bigsize` vs. using `uintN`
```
Comment (Tibo-lg) [inline meeting/meeting_january_11_2022.md:63]: That's a conversation I'd like to have for message representation in general. I think we could remove a lot of implementation vulnerabilities with better typing.

**Comment by Tibo-lg** [comment]
If time would like to discuss this: https://github.com/discreetlogcontracts/dlcspecs/issues/168

File: meeting/meeting_january_11_2022.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(Tibo)**
+* **#Query(Jesse)**
+* **#Query(Ben)**
+* **#Query(Philipp)**
```
Comment (bonomat) [inline meeting/meeting_january_11_2022.md]: Unfortunately I won't be able to make it as I'm in Europe and the call will be at 1am for me.

File: meeting/meeting_january_11_2022.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+## DLC Research
+
+**#Query(Lloyd)**
+
+* BLS Signatures
+  * Can encrypt Bitcoin signatures to these BLS sigs
+    * Breakthrough: Verifiable encryption is possible and efficient for DLCs!?!?!?
+      * Crude verifiable encryption has a batching algorithm that makes it efficient for huge batches
+* Cut and choose sigs do not actually require BLS
+* Idea: OP_CTV also fixes this!?
```
Comment (LLFourn) [inline meeting/meeting_january_11_2022.md]:
```suggestion
* Idea: OP_CTV also fixes this!? ML post: https://mailmanlists.org/pipermail/dlc-dev/2022-January/000102.html
```

**Pull request merged**